stacks-common/src/libcommon.rs

```diff
@@ -80,10 +80,11 @@ pub mod consts {
     pub const PEER_VERSION_EPOCH_2_5: u8 = 0x0a;
     pub const PEER_VERSION_EPOCH_3_0: u8 = 0x0b;
     pub const PEER_VERSION_EPOCH_3_1: u8 = 0x0c;
+    pub const PEER_VERSION_EPOCH_3_2: u8 = 0x0d;
 
     /// this should be updated to the latest network epoch version supported by
     ///  this node. this will be checked by the `validate_epochs()` method.
-    pub const PEER_NETWORK_EPOCH: u32 = PEER_VERSION_EPOCH_3_1 as u32;
+    pub const PEER_NETWORK_EPOCH: u32 = PEER_VERSION_EPOCH_3_2 as u32;
```

Comment:
I think this should be left as 3_1 until the other 3.2 PRs merge to develop. Otherwise, we won't be able to make a release from develop with this PR merged (because those 3.1.0.0.14 releases would "look" like they support the 3.2 hard fork, but won't actually).